about.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+  <link href='https://fonts.googleapis.com/css?family=Lato' rel='stylesheet'>
+  <link rel="stylesheet" href="./style.css">
+  <title>Sundance Film Festival 2023</title>
+</head>
+
+<body>
+  <header id="header">
+    <div class="navbar">
+      <ul class="top-nav">
+        <li><a href="#"><i class="fa fa-facebook-f" aria-hidden="true"></i></a></li>
+        <li><a href="#"><i class="fa fa-twitter" aria-hidden="true"></i></a></li>
+        <li><a href="#">English</a></li>
+        <li><a href="#">My Page</a></li>
+        <li><a href="#">Logout</a></li>
+      </ul>
+      <div class="nav-menu">
+        <i id="close-menu" class="fas fa-times fa-3x" aria-hidden="true"></i>
+        <a href="./index.html"><img class="nav-image" src="./media/institute_logo.png" alt="Logo image"></a>
+        <ul class="about-bottom-nav">
+          <li>
+            <a href="./about.html">About</a>
+          </li>
+          <li>
+            <a href="./index.html#program">Program</a>
+          </li>
+          <li>
+            <a href="./index.html#features">Featured Films</a>
+          </li>
+          <li>
+            <a href="#">Sign up!</a>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </header>
+
+  <main>
+    <i id="open-menu" class="fa fa-bars fa-3x" aria-hidden="true"></i>
+    <section id="about-main">
+      <div class="about-wrapper">
+        <h2>Hello film world!</h2>
+        <h1>Sundance Film Festival 2023</h1>
+        <p class="about-festival">The Sundance Film Festival is the ultimate gathering of original storytellers and audiences seeking new voices and fresh perspectives. Our annual program includes dramatic and documentary features and short films;
+          series and episodic content; and New Frontier, showcasing emerging media in the form of multimedia installations, performances, and films.</p>
+        <p class="contact-text">Please contact us per Email for any further questions</p>
+        <p class="contact-email">festival2023@sundace.com</p>
```

Comment:
- [ ] Please use the same `width` and `height` as defined in the original design :smile: